service/main.go

```diff
@@ -97,6 +97,22 @@ func run(f flags) {
 			ulog.Error(err)
 			os.Exit(1)
 		}
+	case "download-latest":
+		ctx, updater := keybase.NewUpdaterContext(f.appName, f.pathToKeybase, ulog, keybase.CheckPassive)
+		updateAvailable, updateCached, err := updater.CheckAndDownload(ctx)
+		if err != nil {
+			ulog.Error(err)
+			os.Exit(1)
+		}
+		fmt.Println(updateAvailable && updateCached)
```

Comment:
can you drop a comment here and below that the bool output is parsed by the service so we shouldn't change it unless both change